docs/test-cases/vcf-highlight/Conditions/Highlight vcf with simple conditions.md

```diff
@@ -0,0 +1,78 @@
+# Highlighting variations depending on the set conditions
+
+Test verifies
+ - highlighting background of variants in Variants panel and VCF track on Browser in dependence of specified simple conditions in JSON file
```

Comment:
I'd simplify this description a bit: `Background colour of Variations in the Variants panel and VCF track is highlighted according to the selected profile`